goplay/src/main/java/be/tapped/goplay/content/GoPlayApi.kt

```diff
@@ -1,24 +1,32 @@
 package be.tapped.goplay.content
 
-import be.tapped.goplay.common.goPlayApiDefaultOkHttpClient
+import be.tapped.goplay.epg.EpgRepo
+import be.tapped.goplay.epg.httpEpgRepo
+import io.ktor.client.*
+import io.ktor.client.engine.apache.*
+import io.ktor.client.features.json.*
+import io.ktor.client.features.json.serializer.*
```

Comment:
😭 Let's add KtFmt or KtLint in a subsequent PR.
If you don't mind 2-indent, then I propose using KtFmt with `googleStyle()`

Reply:
Yes! 
Feel free to add